### DISCUSSION TITLE
The playback address prompt exceeded the deadline while waiting for connection

### DISCUSSION THREAD

**Discussion by sunzeliang336** [bluenviron/mediamtx#3328] [Questions]
### Question

I tested using ffmpeg transcoding playback on the Windows version to ensure normal playback. After deploying on the Linux system server, the transcoding playback prompt exceeded the deadline while waiting for connection. The IP whitelist has been opened to each other, and the corresponding ports have also been opened. Do you know where the problem lies?[
![Error reporting](https://github.com/bluenviron/mediamtx/assets/169021935/f1af557e-cb62-45f7-a238-5167d42494c1)
](url)[mediamtx_v1.8.1_linux_amd64.tar.gz](https://github.com/bluenviron/mediamtx/releases/download/v1.8.1/mediamtx_v1.8.1_linux_amd64.tar.gz)

**Comment by aler9** [comment]
Hello, the README provides lots of methods to solve WebRTC connectivity issues:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#solving-webrtc-connectivity-issues

[a comment by an automated account is not shown]
